crates/bevy_render/src/primitives/mod.rs

```diff
@@ -4,6 +4,18 @@ use bevy_reflect::Reflect;
 use bevy_utils::HashMap;
 
 /// An axis-aligned bounding box.
```

Comment:
It isn't applied only to `Handle<Mesh>`, people can find what it applies to if they follow the doc of `CalculateBounds`, but maybe I can add a "for example" section?

Reply:
Mentionned it here:

```rust
/// It will be added automatically by the systems in [`CalculateBounds`] to entities that:
/// - could be subject to frustum culling, for example [`Handle<Mesh>`] or `Sprite`,
/// - don't have the [`NoFrustumCulling`] component.
```